### ISSUE TITLE
The name 'bddSetUp' is already defined.

### ISSUE THREAD

**Issue by DevFreeze** [olexale/bdd_widget_test#40]
When defining several `Features` within the .feature file, each with `Background`, then the test will fail, because of bddSetUp being defined several times.

This feature file:

```
Feature: Successful login
  Background:
    Given The app is running

  Scenario:
    Given I'm in the login screen
    And I have valid login credentials
    When I enter my valid credentials
    Then I should see the home screen

Feature: Unsuccessful login
  Background:
    Given The app is running

  Scenario:
    Given I'm in the login screen
    And I don't have valid login credentials
    When I enter my invalid credentials
    Then I should see {'Invalid credentials'} text
```

Will generate this Dart file:

```
void main() {
  Future<void> bddSetUp(WidgetTester tester) async {
    await theAppIsRunning(tester);
  }
  group('''Successful login''', () {
    testWidgets('''''', (tester) async {
      await bddSetUp(tester);
      await imInTheLoginScreen(tester);
      await iHaveValidLoginCredentials(tester);
      await iEnterMyValidCredentials(tester);
      await iShouldSeeTheHomeScreen(tester);
    });
  });
  Future<void> bddSetUp(WidgetTester tester) async {
    await theAppIsRunning(tester);
  }
  group('''Unsuccessful login''', () {
    testWidgets('''''', (tester) async {
      await bddSetUp(tester);
      await imInTheLoginScreen(tester);
      await iDontHaveValidLoginCredentials(tester);
      await iEnterMyInvalidCredentials(tester);
      await iShouldSeeText(tester, 'Invalid credentials');
    });
  });
}
```

Maybe the function bddSetUp() should not be defined outside of the group but inside the group, like this:

```
void main() {
  group('''Successful login''', () {
    Future<void> bddSetUp(WidgetTester tester) async {
        await theAppIsRunning(tester);
    }

    testWidgets('''''', (tester) async {
      await bddSetUp(tester);
      await imInTheLoginScreen(tester);
      await iHaveValidLoginCredentials(tester);
      await iEnterMyValidCredentials(tester);
      await iShouldSeeTheHomeScreen(tester);
    });
  });
  group('''Unsuccessful login''', () {
    Future<void> bddSetUp(WidgetTester tester) async {
        await theAppIsRunning(tester);
    }

    testWidgets('''''', (tester) async {
      await bddSetUp(tester);
      await imInTheLoginScreen(tester);
      await iDontHaveValidLoginCredentials(tester);
      await iEnterMyInvalidCredentials(tester);
      await iShouldSeeText(tester, 'Invalid credentials');
    });
  });
}
```

**Comment by olexale** [comment]
@DevFreeze, the issue should be fixed in `1.5.1`. Do not hesitate to reopen the ticket if you find problems with the implementation.
Thanks for the issue!

**Comment by DevFreeze** [comment]
Thanks for the fast response. Works great :)
